src/protobot_control/src/roboclaw.h

```diff
@@ -0,0 +1,51 @@
+//
+// Created by jacob on 2/7/22.
+//
+
+#ifndef PROTOBOT_ROBOCLAW_H
+#define PROTOBOT_ROBOCLAW_H
+
+#include<iostream>
+#include<cstdlib>
+#include<cstring>
+#include<stdint.h>
+#include<fcntl.h>
+#include<errno.h>
+#include<termios.h>
+#include<unistd.h>
+#include<sys/ioctl.h>
+#include<sys/time.h>
+#include<assert.h>
+#include "protobot.h"
+
+class roboclaw : public protobot {
```

Comment:
shouldn't inherit from protobot, you can use composition and hook/call roboclaw functionality in the control loop in main()